consensus/ibft/verifier.go

```diff
@@ -119,40 +133,39 @@ func (i *backendIBFT) IsValidProposalHash(proposal, hash []byte) bool {
 }
 
 func (i *backendIBFT) IsValidCommittedSeal(
-	proposalHash []byte,
+	proposal []byte,
```

Comment:
If this is still what it was before (Keccak of the RLP'd block - proposalHash), I would preserve the naming and not change it to `proposal`. Just to keep it consistent with other methods in this file.

Reply:
@dbrajovic The code at the latest commit has renamed

Reply:
Reverted at e0a59ada3607352112496a2cb36b6061b2c32a05
@dbrajovic